Config.cmake.in

```diff
@@ -7,6 +7,18 @@ set("@PROJECT_NAME_UPPER@_DOXYGENDOCDIR" "${PACKAGE_PREFIX_DIR}/share/doc/@PROJE
 set("@PROJECT_NAME@_DEPENDENCIES" "@_PACKAGE_CONFIG_DEPENDENCIES_PROJECTS@")
 set("@PROJECT_NAME@_PKG_CONFIG_DEPENDENCIES" "@_PKG_CONFIG_REQUIRES_LIST@")
 
+# Find absolute library paths for all _PKG_CONFIG_LIBS as CMake expects full paths, while pkg-config does not.
+set(@PROJECT_NAME@_LIBRARIES "")
+foreach(lib @_PKG_CONFIG_LIBS@)
+  # If the component is a link directory ("-L/full/path"), skip.
+  string(FIND ${lib} "-L" _is_library_dir)
+  if(${_is_library_dir} EQUAL -1)
+    string(REPLACE "-l" "" lib ${lib})
+    find_library(abs_lib_${lib} ${lib} HINTS ${eigenpy_LIBRARY_DIRS})
```

Comment:
I think `${eigenpy_LIBRARY_DIRS}` is not the correct HINT for a generic implementation.

Reply:
Ah, thanks for noticing! That was a left-over from trying tons of different approaches... I'll get it fixed